nn_sdwan/manifest.json

```diff
@@ -0,0 +1,44 @@
+{
+  "display_name": "Netnology SD-WAN",
+  "maintainer": "info@netnology.io",
+  "manifest_version": "1.0.0",
+  "name": "nn_sdwan",
+  "metric_prefix": "nn_sdwan.",
+  "metric_to_check": [
+    "nn_sdwan.device_control_status",
+    "nn_sdwan.app_aware_routing.latency"
+  ],
+  "creates_events": false,
+  "short_description": "SDWAN Controller Metric Exporter by Netnology",
+  "guid": "a7c79ba9-a4b7-4fa4-b4aa-3868458e2101",
+  "support": "contrib",
+  "supported_os": [
+    "linux",
+    "mac_os",
+    "windows"
+  ],
+  "public_title": "Datadog-Netnology SD-WAN Integration",
+  "categories": [
+    "monitoring",
+    "network"
+  ],
+  "type": "check",
+  "is_public": false,
```

Comment:
Is this intentional, it will need to be `true` when merged to be able to see it within Datadog documentation.

Reply:
Not intentional, maybe that's the default value when the project is created by `ddev`? Either way, fixed.
I also removed `nn_sdwan/images/.gitkeep`.